Course Material/Introduction to Quantum Computing and FiQCI/00 Superposition and measurement.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "qat-category": "1-Circuit generation in Python",
+    "qat-summary": "1-Creation of an EPR pair using two qubits"
+   },
+   "source": [
+    "# Quantum Helloworld: Superposition and measurement\n",
+    "\n",
+    "\n",
+    "The goal of this jupyter notebook is to help you write your first quantum program using Qiskit.\n",
+    "This will help you understand its main functionalities:\n",
+    "\n",
+    "+ Retrieve backend\n",
```

Comment:
Do we assume that all users know how the laptop talks to the quantum computer? It would be nice to have a high-level overview of the execution flow.

Reply:
Could also be included in some lecture notes that are perhaps shown before this notebook.